src/test/java/com/puppycrawl/tools/checkstyle/checks/design/MutableExceptionCheckTest.java

```diff
@@ -19,9 +19,8 @@
 
 package com.puppycrawl.tools.checkstyle.checks.design;
 
+import static com.google.common.truth.Truth.assertWithMessage;
 import static com.puppycrawl.tools.checkstyle.checks.design.MutableExceptionCheck.MSG_KEY;
-import static org.junit.jupiter.api.Assertions.assertArrayEquals;
-import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.fail;
```

Comment:
as discussed here https://github.com/checkstyle/checkstyle/issues/9142#issuecomment-855165188

please replace it with `assertWithMessage("message").fail();`

Reply:
done @timurt

Reply:
@timurt 
```
[ERROR] src\test\java\com\puppycrawl\tools\checkstyle\checks\design\MutableExceptionCheckTest.java:[124,9] (extension) RequireFailForTryCatchInJunit: try/catch either requires a fail at the end of the try clause for junit tests or to remove the try/catch completely.
```

Reply:
Do we remove this check or suppress it?

Reply:
lets suppress it via `sevnt_suppressions.xml`

Reply:
done

Reply:
> [ERROR] src\test\java\com\puppycrawl\tools\checkstyle\checks\design\MutableExceptionCheckTest.java:[124,9] (extension) RequireFailForTryCatchInJunit: try/catch either requires a fail at the end of the try clause for junit tests or to remove the try/catch completely.

This is actually a bug in `RequireFailForTryCatchInJunit`. There is a closed issue for this: https://github.com/sevntu-checkstyle/sevntu.checkstyle/issues/826. I'm going to investigate this issue.